README.md

```diff
@@ -19,7 +19,7 @@
 
 Search UI allows you to quickly implement search experiences without re-inventing the wheel.
 
-It supports **[React](https://reactjs.org)** and **works with any search API**.
+Supports **[React](https://reactjs.org)** and **works Elastic App Search, or any search API**.
```

Comment:
General note, this sounds wishy-washy. "Supports" React, "works with" Elastic App Search, "or" any search API.

What if we could say something more assertive, like:

"A React library for Elastic App Search."

or

"A React library for Elastic App Search that allows you to quickly implement search experiences without re-inventing the wheel"

Just thinking out loud.

Reply:
Agreed! Went with option b, hyperlinking React and App Search.